src/style/cell.ts

```diff
@@ -35,15 +35,12 @@ export const cellFrozen = css`
     position: sticky;
     /* Should have a higher value than 0 to show up above unfrozen cells */
     z-index: 1;
-  }
-`;
-
-export const cellFrozenClassname = `rdg-cell-frozen ${cellFrozen}`;
 
-export const cellFrozenLast = css`
-  @layer rdg.Cell {
-    box-shadow: calc(2px * var(--rdg-sign)) 0 5px -2px rgba(136, 136, 136, 0.3);
+    /* Add box-shadow on the last frozen cell */
+    &:has(+ :not(&)) {
```

Comment:
Does this work if the last frozen cell is also the last cell in the row?

Reply:
It did not work. Fixed with `:nth-last-child`